src/main/java/com/bullhorn/dataloader/task/AbstractTask.java

```diff
@@ -66,4 +76,31 @@ protected void getAndSetBullhornEntityInfo() {
         }
     }
 
+    protected void writeToResultCSV(Result result) {
+        try {
+            csvWriter.writeAttachmentRow(dataMap.values().toArray(new String[0]), result);
+            updateActionTotals(result);
+            updateRowProcessedCounts();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private void updateRowProcessedCounts() {
+        rowProcessedCount.incrementAndGet();
+        if(rowProcessedCount.intValue() % 111 == 0) {
+            printUtil.printAndLog("Processed: " + NumberFormat.getNumberInstance(Locale.US).format(rowProcessedCount) + " records.");
+        }
+    }
+
+    private void updateActionTotals(Result result) {
+        if(result.getAction().equals(Result.Action.INSERT)) {
+            actionTotals.incrementTotalInsert();
+        } else if(result.getAction().equals(Result.Action.UPDATE)){
+            actionTotals.incrementTotalUpdate();
```

Comment:
Please add the `else if` clause for the DELETE action that is in the DeleteTask, for completeness.